src/erlcloud_aws.erl

```diff
@@ -259,6 +259,7 @@ param_list(Values, Key) ->
     [{lists:flatten([Key, $., integer_to_list(I)]), Value} ||
         {I, Value} <- lists:zip(lists:seq(1, length(Values)), Values)].
 
+value_to_string(Float) when is_float(Float) -> io_lib:format("~p", [Float]);
```

Comment:
@waisbrot Initially I was skeptical to `float_to_list`
```erlang
1> float_to_list(4.1).
"4.09999999999999964473e+00"
```
but AWS API handles it pretty well. I modified it as you suggested.